uni/lib/view/common_widgets/horizontal_listview.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class HorizontalWidget extends StatelessWidget {
+  const HorizontalWidget({
+    required this.ListOfWidgets,
+    required this.title,
+    super.key,
+  });
+  final List<Widget> ListOfWidgets;
+  final String title;
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+    appBar: AppBar(
+      title: Text(title),
+      centerTitle: true,
+    ),
+    body: SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      padding: const EdgeInsets.all(12),
+      child: Row(
+      children: ListOfWidgets,
+    ),)
+  );
```

Comment:
The goal of your issue is to "scale" the first child to about 90% of the width (see the mockups).